packages/@lightningjs/ui-components/src/components/Tile/Tile.mdx

```diff
@@ -90,16 +90,16 @@ class Basic extends lng.Component {
+| name               | type    | required | default    | description                                                                                                                                                                                                                             |
+| ------------------ | ------- | -------- | ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| artwork            | object  | false    | undefined  | Object containing all properties supported in the [Artwork component](?path=/docs/components-artwork--artwork)                                                                                                                          |
+| badge              | object  | false    | undefined  | Object containing all properties supported in the [Badge component](?path=/docs/components-badge--text)                                                                                                                                 |
+| checkbox           | object  | false    | undefined  | Object containing all properties supported in the [Checkbox component](?path=/docs/utilities-checkbox--checkbox)                                                                                                                        |
+| label              | object  | false    | undefined  | Object containing all properties supported in the [Label component](?path=/docs/components-label--label)                                                                                                                                |
+| metadata           | object  | false    | undefined  | Object containing all properties supported in the [MetadataTile component](?path=/docs/components-metadatatile--metadata-tile)<br /> Can use a different Metadata component by passing in a 'type' and then that component's properties |
+| metadataLocation   | string  | false    | 'standard' | Controls where there metadata is displayed in relation to the Tile. Available values are 'standard' and 'inset'                                                                                                                         |
+| persistentMetadata | boolean | false    | false      | Metadata will be shown at all times if set to true, otherwise it will only show when the Tile has focus                                                                                                                                 |
+| progressBar        | object  | false    | undefined  | Object containing all properties supported in the [ProgressBar component](??path=/docs/utilities-progressbar--progress-bar)                                                                                                             |
```

Comment:
We have to change path for progressBar and Checkbox, they are pointed to utilities here.